docs/reference/index-modules/merge.asciidoc

```diff
@@ -150,6 +151,7 @@ Defaults to unbounded.
 [float]
 [[log-doc]]
 ==== log_doc
+deprecated[1.6.0, this policy will be removed in 2.0 in favor of the [tiered] policy]
```

Comment:
deprecated[1.6.0,This policy will be removed in 2.0 in favour of the <<tiered,tiered>> merge policy]